src/Interpreters/executeQuery.cpp

```diff
@@ -197,6 +197,7 @@ static void onExceptionBeforeStart(const String & query_for_logging, Context & c
     elem.type = QueryLogElementType::EXCEPTION_BEFORE_START;
 
     elem.event_time = current_time;
+    elem.event_time_microseconds = current_time_microseconds;
```

Comment:
Seconds part may be different (see https://github.com/ClickHouse/ClickHouse/blob/44e168bea784f22832384588dea7c317d490b474/src/Interpreters/executeQuery.cpp#L254-L255)

Reply:
Added a comment that the caller to this function will always construct both the times from same timespec, and also implemented the same in places where this is being called.